lib/screens/group_notes_widget.dart

```diff
@@ -62,12 +72,11 @@ class _GroupNotesWidgetState extends State<GroupNotesWidget> {
         title: const Text('List of notes'),
         actions: [
           IconButton(
-              onPressed: () {
-                //context.read<ProviderData>().getNotesData();
-                refresh();
-              },
-              icon: const Icon(Icons.refresh),),
-          //_logOutButton(),
+            onPressed: () {
+              refresh();
+            },
```

Comment:
Functions can be passed around as values, like so.
This expression is equivalent to yours, without an extra closure.
```suggestion
            onPressed: refresh,
```